README.md

```diff
@@ -60,30 +60,30 @@ Read the [API documentation](https://doc.esdoc.org/github.com/Kinto/kinto-client
 
 In the browser, you can load prebuilt scripts hosted on npmcdn:
 
-- [kinto-client.js](https://npmcdn.com/kinto-client/dist/kinto-client.js)
-- [kinto-client.min.js](https://npmcdn.com/kinto-client/dist/kinto-client.min.js)
-- [kinto-client.noshim.js](https://npmcdn.com/kinto-client/dist/kinto-client.noshim.js)
+- [kinto-http.js](https://npmcdn.com/kinto-http.js/dist/kinto-http.js)
+- [kinto-http.min.js](https://npmcdn.com/kinto-http.js/dist/kinto-http.min.js)
+- [kinto-http.noshim.js](https://npmcdn.com/kinto-http.js/dist/kinto-http.noshim.js)
 
 ```html
-<script src="https://npmcdn.com/kinto-client/dist/kinto-client.min.js"></script>
+<script src="https://npmcdn.com/kinto-http.js/dist/kinto-http.min.js"></script>
 ```
 
 In nodejs:
 
 ```
-$ npm install kinto-client --save
+$ npm install kinto-http --save
 ```
 
 Then (ES6):
 
 ```js
-import KintoClient from "kinto-client";
+import KintoClient from "kinto-http.js";
```

Comment:
3 lines above you installed it using "$ npm install kinto-http --save" so it's `import KintoClient from "kinto-http"`